udata_hydra/config_default.toml

```diff
@@ -5,6 +5,7 @@ SENTRY_DSN = ""
 TESTING = false
 # max postgres pool size
 MAX_POOL_SIZE = 50
+USER_AGENT = "udata-hydra/1.0"
```

Comment:
We should override that in prod and [add a comment](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax) to contact us in case of trouble